src/Component/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styled from "styled-components";
+import { AvatarCompSize, AvatarImgSize } from "../../variables";
```

Comment:
```suggestion
import { AvatarCompSize, AvatarImgSize } from "../../constants";
```
I am assuming the avatar size does not change so the file should be called constants

Reply:
yeah, will remember